src/taskpane/server.js

```diff
@@ -761,8 +877,10 @@ function putArtifactToSpira(entry, user, projectId, artifactTypeId, parentId) {
     // TEST CASES
     case ART_ENUMS.testCases:
       putUrl = API_PROJECT_BASE + projectId + '/test-cases?';
-      entry.TestSteps = null;
-      JSON_body = JSON.stringify(entry);
+      if (!IS_GOOGLE) {

```

Comment:
If this code block only applies to Excel, how does google handle test cases / steps?

Reply:
It handles using the "case.TestStep" in this same function. If I'm not wrong, I had to change it because the Excel TS object was malformed and Google wasn't. Just to be safe, I tested updating TC+TS for Google and Excel. Both are working fine.